adafruit_hue.py

```diff
@@ -38,7 +43,12 @@ class Bridge:
     HTTP Interface for interacting with a Philips Hue Bridge.
     """
 
-    def __init__(self, wifi_manager, bridge_ip=None, username=None):
+    def __init__(
+        self,
+        wifi_manager: Any,
```

Comment:
I just recently added a typing protocol for this!  Add `adafruit-circuitpython-typing` to `requirements.txt` and add `from circuitpython_typing.http import HTTPProtocol` to the `typing` import block.  Then you can type `wifi_manager` as `HTTPProtocol`!